### PR TITLE
Add premonition support for admonitions

### PR DESCRIPTION
**Pull request by iranzo**
**What this PR does / why we need it**:

Check example at https://deploy-preview-348--kubevirtio.netlify.com/2019/kubevirt-cr-condition-types-rename

Under `what does it mean to us`

**Does this PR fix any issue?** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
> Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by iranzo** [comment]
imagen

File: _sass/_premonition.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  .fa {
+    font-size: 18px;
+    opacity: .3;
+    padding-top: 2px;
+    padding-left: 20px;
+  }
+svg {
+margin:13px 0 0 15px;
```
Comment (matthewcarleton) [inline _sass/_premonition.scss]:
```suggestion
margin:4px 0 0 15px;
```

**Comment by iranzo** [comment]
imagen

File: _sass/_premonition.scss [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  .fa {
+    font-size: 18px;
+    opacity: .3;
+    padding-top: 2px;
+    padding-left: 20px;
+  }
+svg {
+margin:11px 0 0 15px;
```
Comment (matthewcarleton) [inline _sass/_premonition.scss]:
```suggestion
margin:4px 0 0 15px;
```

File: _sass/_premonition.scss [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+  code {
+    background-color: #fff;
+    color: $default-color;
+  }
+
+  .header {
+    font-weight: 700;
+    font-size: 15px;
+    color: $default-color;
```
Comment (matthewcarleton) [inline _sass/_premonition.scss]:
```suggestion
    color: $default-color;
    margin-bottom: 5px;
```

**Comment by iranzo** [comment]
/hold

Seems that plugin for admonitons can't be enabled on GH Pages but it works on netlify fine

**Pull request merged**